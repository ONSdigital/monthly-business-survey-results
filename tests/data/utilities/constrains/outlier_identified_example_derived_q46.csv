reference,period,spp_form_id,questioncode,value,outlier_weight
1,202001,15,46,100,1
1,202001,15,40,100,1
