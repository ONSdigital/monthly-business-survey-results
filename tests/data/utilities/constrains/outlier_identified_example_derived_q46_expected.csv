reference,period,spp_form_id,questioncode,value,outlier_weight,winsorised_value,post_winsorised
1,202001,15,46,100,1,100,FALSE
1,202001,15,40,100,1,100,FALSE
